django_mongodb/features.py

```diff
@@ -34,10 +34,21 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         "basic.tests.SelectOnSaveTests.test_select_on_save_lying_update",
         # Order by constant not supported:
         # AttributeError: 'Field' object has no attribute 'model'
+        "aggregation.tests.AggregateTestCase.test_annotate_values_list",
+        "aggregation.tests.AggregateTestCase.test_combine_different_types",
+        "aggregation.tests.AggregateTestCase.test_grouped_annotation_in_group_by",
+        "aggregation.tests.AggregateTestCase.test_non_grouped_annotation_not_in_group_by",
+        "aggregation.tests.AggregateTestCase.test_values_annotation_with_expression",
+        "aggregation.tests.AggregateTestCase.test_annotate_ordering",
```

Comment:
cool, but I think we should open a issue for this kind of error, I put them together because think the bad ordering and the crash could be addressed in the same way.